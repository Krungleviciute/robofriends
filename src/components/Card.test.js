import React from "react";
import Card from "./Card";
import { create } from "react-test-renderer";

describe("Card component", () => {
  it("should check if Card component renders", () => {
    const card = create(<Card />);
    expect(card.toJSON()).toMatchSnapshot();
  });
});
