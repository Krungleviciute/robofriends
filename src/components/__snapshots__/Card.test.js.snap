// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should check if Card component renders 1`] = `
<div
  className="tc bg-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robo profile"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
